cognito/requirements.txt

```diff
@@ -0,0 +1,18 @@
+boto3==1.9.6
+botocore==1.12.6
+certifi==2018.8.24
+chardet==3.0.4
+ecdsa==0.13
+envs==1.2.6
+future==0.16.0
+futures==3.2.0
+idna==2.7
+jmespath==0.9.3
+pycryptodome==3.3.1
+python-dateutil==2.7.3
+python-jose-cryptodome==1.3.2
+requests==2.9.1
+six==1.11.0
+socotratools==1.3.63
```

Comment:
socotraltools 1.3.63? socotratools versions:  (from versions: 1.3, 1.3.1, 1.3.4, 1.3.5, 1.3.6

Reply:
Updated to 1.3.7